src/events/interactionCreate.ts

```diff
@@ -12,7 +12,7 @@ export default {
       if (!bypass) return
       try {
         console.log(
-          `[Command] ${interaction.user.tag} (${interaction.user.id}) > ${interaction.commandName}`
+          `[Command] ${interaction.user.tag} (${interaction.user.id}) > ${interaction.commandName}`,
```

Comment:
Should trailing comma be `es5`. Feel like too much comma here.

Reply:
cc: @beam41

Reply:
It fixes problem of someone change function signature for me, I admit it's too much sometimes 😅

Reply:
Btw, prettier automatically add comma for me so I don't think about it much.